client/src/app/shared/components/slide-container/slide-container.component.ts

```diff
@@ -137,7 +138,7 @@ export class SlideContainerComponent extends BaseComponent {
     private updateScroll(): void {
         if (this.slideOptions.scrollable) {
             let value = this.scroll;
-            value *= -50;
+            value *= -100;
```

Comment:
Please undo the swapping of "Down" and "Up" and do this here.

Reply:
I used better comments in projector detail component html.